test/API.spec.ts

```diff
@@ -16,6 +39,18 @@ function removeVariableProperties(obj: any) {
   return obj;
 }
 
+// Import the actual Response object from node-fetch, rather than a mocked version.
+// See https://jestjs.io/docs/bypassing-module-mocks for more information.
+const {Response} = jest.requireActual('node-fetch');
+
+// Mock node-fetch so we can call it without actually making a network request.
+jest.mock('node-fetch');
+
+const makeNodeFetchReturn = (returnVal: any): void => {
```

Comment:
While `any` and `@ts-ignore` are convenient, there's usually never a good reason to use it. Jest provides docs on mocks with TS:
https://jestjs.io/docs/mock-function-api#typescript.

Also the fn name `makeNodeFetchReturn` can be improved. You could try `mockResponse`.

Reply:
Okay, thank you for the link. I believe 98fc8a7 addresses this, and this pull request is ready for another review now.